docs: add warning for workspace restarts on automount changes (CRW-9859) (#3025)
codenvy/che-docs: modules/end-user-guide/pages/mounting-configmaps.adoc

* In your user {orch-namespace}, you created a new ConfigMap or determined an existing ConfigMap to mount to all `{devworkspace}` containers.

[WARNING]
====

> Applying a Secret or ConfigMap with the `controller.devfile.io/mount-to-devworkspace: 'true'` label (or modifying an existing one) triggers an immediate restart of all running workspaces in that {namespace} by applying the configuration. Ensure users have saved their work before applying these changes.

Reviewer: ```suggestion
Applying or modifying a `Secret` or `ConfigMap` with the `controller.devfile.io/mount-to-devworkspace: 'true'` label restarts all running workspaces in the namespace. Ensure that users save their work before you apply these changes.
```
